db/re/mob_db.txt

```diff
@@ -2649,20 +2649,23 @@
+// Dummy (inaccurate)
+3495,DR_EGGRING,Eggring,Eggring,1,60,1,27,20,1,8,9,2,5,6,1,1,0,6,5,10,12,1,3,21,0x83,400,1872,672,480,0,0,0,0,0,0,0,909,7000,1202,100,938,400,512,1000,713,1500,512,150,619,20,0,0,0,0,4001,1
+3496,DR_LUNATIC,Leaf Lunatic,Leaf Lunatic,3,55,1,36,27,1,11,12,18,0,10,3,3,0,8,5,10,12,0,2,60,0x81,200,1456,456,336,0,0,0,0,0,0,0,705,6500,949,1000,2262,4,512,2000,507,600,515,1100,622,20,0,0,0,0,4006,1
+3497,DR_FABRE,Grass Fabre,Grass Fabre,6,72,1,54,41,1,12,15,24,0,12,5,5,5,12,5,10,12,0,4,22,0x2000081,400,1672,672,480,0,0,0,0,0,0,0,914,6500,949,500,1502,80,721,5,511,700,705,1000,1501,200,0,0,0,0,4002,1
+3498,DR_HORNET,Wild Hornet,Wild Hornet,11,90,1,81,60,1,13,16,7,1,12,24,4,5,6,5,10,12,0,4,24,0x2001081,150,1292,792,216,0,0,0,0,0,0,0,992,80,939,9000,909,3500,1208,15,511,350,518,150,0,0,0,0,0,0,4019,1
+3499,DR_RODA_FROG,Sweet Roda Frog,Sweet Roda Frog,13,160,1,90,68,1,19,23,12,5,12,6,4,0,14,9,10,12,1,5,21,0x81,200,2016,816,288,0,0,0,0,0,0,0,918,9000,908,500,511,300,721,7,713,2000,0,0,0,0,0,0,0,0,4014,1
+3500,DR_DESERT_WOLF_B,Hunter Desert Wolf,Hunter Desert Wolf,45,1219,1,342,386,1,80,91,55,3,27,24,30,15,33,5,10,12,1,2,22,0x1089,200,1054,504,432,0,0,0,0,0,0,0,1011,20,920,9000,2308,10,517,650,528,1050,919,5500,578,600,0,0,0,0,4029,1
+3501,DR_SPORE,Trans Spore,Trans Spore,18,280,1,117,87,1,25,33,12,10,15,5,10,0,12,0,10,12,1,3,21,0x81,200,1872,672,288,0,0,0,0,0,0,0,921,9000,507,800,510,50,743,10,2220,40,7033,5,578,600,0,0,0,0,4022,1
+3502,DR_BASILISK1,Scout Basilisk,Scout Basilisk,17,234,1,117,87,1,38,50,20,3,15,8,17,0,15,0,10,12,1,1,29,0x3885,400,2612,912,288,0,0,0,0,0,0,0,957,9000,724,5,938,1000,958,50,727,70,0,0,0,0,0,0,0,0,4038,1
+3503,DR_BASILISK2,Combat Basilisk,Combat Basilisk,61,2614,1,583,656,1,216,245,78,5,56,12,19,11,27,10,10,12,1,1,49,0x3885,250,2456,912,504,0,0,0,0,0,0,0,958,6000,756,110,509,700,511,800,2609,60,934,150,1260,1,0,0,0,0,4110,1
+3504,DR_BASILISK3,Combat Basilisk,Combat Basilisk,61,2614,1,583,656,1,216,245,78,5,56,12,19,11,27,10,10,12,1,1,49,0x3885,250,2456,912,504,0,0,0,0,0,0,0,958,6000,756,110,509,700,511,800,2609,60,934,150,1260,1,0,0,0,0,4110,1
+3505,DR_BIG_EGGRING,Giant Eggring,Giant Eggring,13,156,1,97,73,4,20,23,13,2,12,3,5,5,10,5,10,12,1,3,62,0x84,1000,1768,768,576,0,0,0,0,0,0,0,993,50,905,9000,1405,30,511,350,711,300,706,3,1967,10,0,0,0,0,4030,1
+3506,DR_MANDRAGORA,Jungle Madragora,Jungle Madragora,42,1260,1,315,354,1,80,120,48,31,35,21,41,12,29,60,10,12,1,3,21,0x6203695,300,1072,672,480,0,0,0,0,0,0,0,2257,200,619,50,722,1000,2116,1000,7938,500,512,8000,531,4000,0,0,0,0,4197,1
+3507,DR_POM_SPIDER,Fruits Pom Spider,Fruits Pom Spider,145,122110,1,8010,8037,1,632,847,240,115,156,85,178,88,139,102,10,12,1,4,62,0x2003885,250,864,1056,576,0,0,0,0,0,0,0,12573,500,11520,1000,1042,1000,13431,10,1654,3,0,0,0,0,0,0,0,0,4514,1
+3508,DR_EGGRING_G,Eggring,Eggring,1,60,1,27,20,1,8,9,2,5,6,1,1,0,6,5,10,12,1,3,21,0x83,400,1872,672,480,0,0,0,0,0,0,0,909,7000,1202,100,938,400,512,1000,713,1500,512,150,619,20,0,0,0,0,4001,1
```

Comment:
No info about card drop from DP, but 4001 sounds wrong due to the other monsters.